webhook/logging/logging.go

```diff
@@ -31,11 +33,15 @@ func WebhookLogger(errorType string, errorMessage error) error {
 
 	file, err := os.OpenFile(logFileName, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
 	if err != nil {
+		fmt.Print("fmt: \n", err)
 		return err
 	}
 	defer file.Close()
 
+	multi := io.MultiWriter(os.Stdout)
```

Comment:
@KayZhang52 I think you should add the file here too because we want to log also in the file not only on the standard output